hydrolib/core/dflowfm/mdu/models.py

```diff
@@ -1024,6 +1174,18 @@ class Comments(INIBasedModel.Comments):
             "Write water level gradient to map file (only for 1D links) (1:yes, 0:no).",
             alias="wrimap_water_level_gradient",
         )
+        wrimap_tidal_potential: Optional[str] = Field(
+            "Write tidal potential to map file (1: yes, 0: no)",
+            alias="wrimap_tidal_potential",
+        )
+        wrimap_sal_potential: Optional[str] = Field(
+            "Write self attraction and loading potential to map file (1: yes, 0: no)",
+            alias="wrimap_SAL_potential",
+        )
+        wrimap_internal_tides_dissipation: Optional[str] = Field(
+            "Write internal tides dissipation to map file (1: yes, 0: no)",
+            alias="wrimap_internal_tides_dissipation",
+        )
```

Comment:
I am missing the keyword `wrimap_waterdepth` and `Wrimap_velocity_magnitude `

Reply:
Added